src/components/metadata/metadata-code.test.js

```diff
@@ -49,57 +48,3 @@ describe('MetaDataCode', () => {
     ).toBe(true);
   });
 });
-
-describe('MetaCodeToggle', () => {
-  const input = (wrapper) =>
-    wrapper.find('.pipeline-metadata__code-toggle-input');
```

Comment:
As I mentioned in the comment above, the css class should be updated to reflect the new css rather than the entire test suite being deleted.

Reply:
@rashidakanchwala following up on my comment, the quickest way to tackle this is to update the css class here to reflect the new classes that were set up under `toggle.js` :) Thank you

Reply:
resolving this as per our discussion in our 1:1. Thanks @rashidakanchwala